plugins/search-backend/config.d.ts

```diff
@@ -17,6 +17,11 @@
 export interface Config {
   /** Configuration options for the search plugin */
   search?: {
+    /**
+     * Sets the maximum max page limit size. Defaults to 100.
+     */
+    maxPageLimit?: number;
```

Comment:
Does it make sense to configure the default page limit value as well? It is hardcoded to 25 by all search engines, as follows:

https://github.com/backstage/backstage/blob/5cc6d6ce0541fc3a792be2f48caf2f8f43210fc5/plugins/search-backend-module-elasticsearch/src/engines/ElasticSearchSearchEngine.ts#L213

https://github.com/backstage/backstage/blob/5cc6d6ce0541fc3a792be2f48caf2f8f43210fc5/plugins/search-backend-module-pg/src/PgSearchEngine/PgSearchEngine.ts#L140

https://github.com/backstage/backstage/blob/5cc6d6ce0541fc3a792be2f48caf2f8f43210fc5/plugins/search-backend-node/src/engines/LunrSearchEngine.ts#L76

Reply:
ah yeah that's a good idea. I'll make the appropriate updates to expose that option too

Reply:
@camilaibs actually I'll do that in a follow up PR if you don't mind. I'm current OOO without my laptop for the week and am hoping to have this in for the next release 🙂

Reply:
Sounds good 🙌🏻